### PR TITLE
system tests coverage per service/module

### PR DESCRIPTION
**Pull request by skudasov**
Allows us to automatically collect/reset coverage of system-level tests per service/module.
Can be used as a helpful debug tool to find some non-covered places.
Check `TUTORIAL.md` updates.

### CODE REVIEW

File: pkg/cdk8s/goc/goc.go [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+					},
+				},
+			},
+		})
+}
+
+func container(vars vars) *k8s.Container {
+	return &k8s.Container{
+		Name:            a.Str(vars.BaseName),
+		Image:           a.Str("f4hrenh9it/goc-target:latest"),
```
Comment (tateexon) [inline pkg/cdk8s/goc/goc.go:122]: Where is the dockerfile for this? Do we have a ticket to get this docker image auto built on changes somewhere?

File: pkg/cdk8s/http_dummy/dummy.go [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+					},
+				},
+			},
+		})
+}
+
+func container(vars vars) *k8s.Container {
+	return &k8s.Container{
+		Name:            a.Str(vars.BaseName),
+		Image:           a.Str("f4hrenh9it/goc:latest"),
```
Comment (tateexon) [inline pkg/cdk8s/http_dummy/dummy.go:138]: Where is the dockerfile for this? Do we have a ticket to get this docker image auto built on changes somewhere?

**Review by tateexon — CHANGES_REQUESTED** [review]

File: examples/coverage/env.go [hunk 3]
```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"time"
+
+	"github.com/smartcontractkit/chainlink-env/environment"
+	dummy "github.com/smartcontractkit/chainlink-env/pkg/cdk8s/goc"
```
Comment (sebawo) [inline examples/coverage/env.go:7]:
```suggestion
	goc "github.com/smartcontractkit/chainlink-env/pkg/cdk8s/goc"
```

File: examples/coverage/env.go [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+	goc "github.com/smartcontractkit/chainlink-env/pkg/cdk8s/http_dummy"
```
Comment (sebawo) [inline examples/coverage/env.go:8]:
```suggestion
	dummy "github.com/smartcontractkit/chainlink-env/pkg/cdk8s/http_dummy"
```

**Review by sebawo — APPROVED** [review]

**Review by tateexon — APPROVED** [review]

**Review by AnieeG — APPROVED** [review]

**Pull request merged**